src/main/java/com/google/sps/servlets/UsersServlet.java

```diff
@@ -13,43 +13,30 @@
 // limitations under the License.
 
 package com.google.sps.servlets;
-import java.io.IOException;
+
 import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
 import com.google.appengine.api.datastore.Entity;
 import com.google.appengine.api.datastore.PreparedQuery;
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.api.datastore.Query.SortDirection;
 import com.google.gson.Gson;
-import java.util.HashMap;
-import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-import java.util.ArrayList;
-import java.util.List;
-import com.google.datastore.v1.PropertyFilter;
+import org.apache.commons.io.IOUtils;
```

Comment:
what does IOUtils do

Reply:
It allows us to accept JSON objects as input to the servlet, sent through the request body